### DISCUSSION TITLE
Broken attachments with programs (zipped exe files)

### DISCUSSION THREAD

**Discussion by darnook** [PrestaShop/PrestaShop#31760] [Q&A]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Recently found that after upload, some attachments with programs (zipped exe files) are broken after download.
What can cause that behavior?

Raw PDF files are okay.

### Expected behavior

.

### Steps to reproduce

.

### PrestaShop version(s) where the bug happened

1.7.8.8

### PHP version(s) where the bug happened

7.4.33

**Comment by matks** [comment]
Hi @darnook I transformed this issue into Q&A as it is not a bug report

I dont think PrestaShop modifies your files however I suggest you check your webserver (apache or nginx probably) configuration, it might be altering your attachments

**Reply by darnook** [reply]
Best part... download uploaded file from BO - works, download from FO brokes downloaded file...
Also fresh install of PS gives good attachments both in BO and FO...
What the h i happend....
